### PR TITLE
Add legacy sections

### PR DESCRIPTION
**Pull request by j0ntsang**

### CODE REVIEW

File: src/styles/_base_lists.scss [hunk 1]
```diff
@@ -8,28 +8,7 @@ li {
   padding: 0;
 }
 
-[class*="__list"],
-[class*="__menu"],
-[class*="__nav"] {
+ul,
+li {
```
Comment (leonardofaria) [inline src/styles/_base_lists.scss]: Should not be `ul, ol` here?
`<li>` elements don't need a `list-style` property.

**Review by leonardofaria — APPROVED** [review]
Looks solid @j0ntsang 
I only have one question about CSS.

File: src/styles/_base_lists.scss (hunk 1 above)
Comment (j0ntsang) [reply src/styles/_base_lists.scss]: Hot-diggity-dog, good catch!

**Review by erin-bush — APPROVED** [review]
Looks okay to me!

**Review by josephajose — APPROVED** [review]
@j0ntsang Looks cool to me from a logical perspective.

**Pull request merged**